themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md

```diff
@@ -565,33 +709,41 @@ final var protocol = config.require("protocol");
 
 {{% choosable language typescript %}}
 
-We also need to create `Container` resources for the frontend and mongo
-containers. Put the `mongoContainer` declaration just above the `backendContainer` one, and the
-`frontendContainer` declaration at the end of the file. Here's the code for the mongo container:
+We also need to create `Container` resources for the frontend and Mongo
+containers. Put the `mongoContainer` declaration just before the `backendContainer` one, and the
+`frontendContainer` declaration after the `backendContainer` declaration. Here's the code for the Mongo container:
```

Comment:
Nit: There isn't a `frontendContainer` declaration in the code as such; it's named `_` in the code ('cause Go), so we might want to adjust the wording here somewhat. Up to you.

Reply:
I've reworded this to account for Go and throwing the value away.